modules/metering-debugging.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-troubleshooting-debugging.adoc
+
+[id="metering-debugging_{context}"]
+
+= Debugging metering
+Debugging metering is much easier when you interact directly with the various components. The sections below detail how you can connect and query Presto and Hive, as well as view the dashboards of the Presto and HDFS components.
+
+[NOTE]
+====
+All of the commands in this section assume you have installed metering through OperatorHub in the `openshift-metering` namespace.
+====
+
+== Get reporting operator logs
+The command below will follow the logs of the `reporting-operator`.
+
+[source,bash]
+----
+$ kubectl -n openshift-metering logs "$(kubectl -n openshift-metering get pods -l app=reporting-operator -o name | cut -c 5-)" -c reporting-operator
```

Comment:
@lamek if you want to follow the logs you need to add `-f` flag after `kubectl -n openshift-metering logs`

Reply:
`$ kubectl -n openshift-metering logs -f "$(kubectl -n openshift-metering get pods -l app=reporting-operator -o name | cut -c 5-)" -c reporting-operator`
@pruan-rht does this look good?

Reply:
@lamek, yes that looks good now (you still need to actually update the PR)